README.md

```diff
@@ -1,2 +1,50 @@
-# playground-for-php
-Work in progress.
+# Appwrite's Php Playground 🎮
```

Comment:
`Php` should be **PHP**. Should be fixed across all appearances.  

`AppWrite` should be **Appwrite**. Should be fixed across all appearances.

Reply:
Hopefully this are the last changes before we can merge 🤞🏻

Reply:
@eldadfux check the changes